DependencyInjection/CmfSeoExtension.php

```diff
@@ -252,29 +258,51 @@ private function loadSitemapHandling($config, XmlFileLoader $loader, ContainerBu
         $loader->load('sitemap.xml');
 
         $configurations = $config['configurations'];
-        // if there are no explicit configurations, enable the default sitemap
-        if (!count($configurations)) {
-            $configurations['sitemap'] = array();
+
+        $helperStatus = array();
+        foreach ($this->sitemapHelperMap as $helper => $tag) {
+            $helperStatus[$helper] = array();
+            $serviceDefinitionIds = $container->findTaggedServiceIds($tag);
+            foreach ($serviceDefinitionIds as $id => $attributes) {
+                if (0 === strncmp($this->getAlias(), $id, strlen($this->getAlias()))) {
+                    // avoid interfering with services that are not part of this bundle
```

Comment:
if we don't do this, behaviour would depend on order in which bundles are instantiated